MapboxDirections/MBIntersection.swift

```diff
@@ -50,6 +50,8 @@ public class Intersection: NSObject, NSSecureCoding {
      */
     public let usableApproachLanes: IndexSet?
     
+    public let roadClasses: [RoadClass]
```

Comment:
An array of enumeration values won’t bridge very cleanly to Objective-C. Additionally, the order of classes doesn’t matter. So we should turn the RoadClass enumeration into a RoadClasses option set (implemented using `NS_OPTION` in Objective-C), then change this property’s type to simply RoadClasses.